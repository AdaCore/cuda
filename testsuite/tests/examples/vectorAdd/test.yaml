driver: examples
input_directory: "0_Introduction/vectorAdd"
