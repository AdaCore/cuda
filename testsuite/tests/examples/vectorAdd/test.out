[Vector addition of  4096 elements]
CUDA kernel launch with  16 blocks of  256  threads
Copy output data from the CUDA device to the host memory
Test PASSED
Done
